File: 800/java/734A_AntonAndDanik.java
import java.io.*;
import java.util.*;

class Solution {
	public static void main (String args[]) {
		Scanner sc = new Scanner (System.in);
		int n = sc.nextInt(), anton = 0, danik = 0;
		String s = sc.next();
		for (int i = 0; i < s.length(); i++) {
			if (s.charAt(i) == 'A') {
				anton++;
			}
			if (s.charAt(i) == 'D') {
				danik++;
			}
		}
		if (anton > danik) {
			System.out.println("Anton");
		} else if (danik > anton) {
			System.out.println("Danik");
		} else {
			System.out.println("Friendship");
		}
	}
}

File: 800/cpp/734A_AntonAndDanik.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	int n, a = 0, d = 0;
	char c[1];
	cin >> n;
	while (n--) {
		cin >> c[0];
		if (c[0] == 'A') {
			a++;		
		} else {
			d++;
		}
	}
	if (a > d) {
		cout << "Anton\n";
	} else if (d > a) {
		cout << "Danik\n";
	} else {
		cout << "Friendship\n";
	}
}